src/tools/rbd_mirror/CMakeLists.txt

```diff
@@ -1,10 +1,12 @@
 add_library(rbd_mirror_types STATIC
+  Types.cc
```

Comment:
The `DWARF error: invalid or unhandled FORM value: 0x23` link failure in Jenkins might be caused by `Types.cc` being added both to `rbd_mirror_types` and `rbd_mirror_internal` below.

Reply:
Have removed one from `rbd_mirror_types` and gave it a build with gcc tool-chain. The build succeeds pretty fine. Will push the same to test it with clang on jenkins.


Thank you @idryomov